File: api/src/main/java/com/noxcrew/noxesium/core/network/clientbound/ClientboundCustomSoundStopPacket.java
package com.noxcrew.noxesium.core.network.clientbound;

import com.noxcrew.noxesium.api.network.NoxesiumPacket;

/**
 * Sent by a server to stop a custom sound by its id.
 */
public record ClientboundCustomSoundStopPacket(int id) implements NoxesiumPacket {}

File: paper/legacy/src/main/kotlin/com/noxcrew/noxesium/legacy/paper/api/network/clientbound/ClientboundCustomSoundStopPacket.kt
package com.noxcrew.noxesium.legacy.paper.api.network.clientbound

import com.noxcrew.noxesium.legacy.paper.api.network.NoxesiumPacket
import com.noxcrew.noxesium.legacy.paper.api.network.NoxesiumPackets

/**
 * Sent by a server to stop a custom Noxesium sound by its id.
 */
public data class ClientboundCustomSoundStopPacket(
    public val id: Int,
) : NoxesiumPacket(NoxesiumPackets.CLIENT_STOP_SOUND)
